Guidelines.md

```diff
@@ -1360,7 +1357,7 @@ POST https://api.contoso.com/v1.0/databases/
 
 ```http
 HTTP/1.1 202 Accepted
-Operation-Location: https://api.contoso.com/v1.0/operations/123
+Location: https://api.contoso.com/v1.0/operations/123
```

Comment:
This is a breaking change. While I like the use of a standard header here, I don't think it meets the bar for a breaking change. Stability is a feature, in and of itself. 

I know we considered the location header at the time we wrote these, but I don't recall why it was rejected.

Reply:
Isn't this part of why we adopted semantic versioning for the guidelines? Putting stability as the guiding priority means that we will grind forward progress to a halt. This is a positive change in the guidelines. It warrants a bump to version 3. As before, APIs are not required to make breaking changes just so they can adopt the new guidelines - but for new APIs, wouldn't we want the right baseline in place?

In my opinion, we should be putting additive changes into 2.x of the API guidelines, but not unnecessarily blocking a 3.x of the guidelines from being created. We *could* say that we haven't gathered sufficient breaking changes to warrant a 3.x at this point, but then I would ask where we plan to track those changes, and whether we can start on 3.x-preview1 even if we choose not to release 3.x until we do get to critical mass.